_posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+author: grassycheetah94
+title: "Thomas' Turtlehack"
+---
+
+Here is the program I am embedding:
+<iframe src="https://trinket.io/embed/python/2a682241b6" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+---
+Reflection:
+I wanted to create a turtle version of the ‘Hello Chuck’ assignment we had this weekend. I started with creating a ‘man’. This took a lot longer than I originally thought it would, so he is not as detailed as I wanted him to be.  But the basic concept is still there. As you can see below.
+
+```
+#Mouth
+jack.goto(25,15)
+jack.pendown()
+jack.goto(20,10)
+jack.goto(-20,10)
+jack.goto(-25,15)
+jack.penup()
+#Nose
+jack.goto(7,35)
+jack.pendown()
+jack.goto(5,33)
+jack.goto(3,35)
+jack.goto(0,30)
+jack.goto(-3,35)
+jack.goto(-5,33)
+jack.goto(-7,35)
+jack.penup()
+#Eyes
+jack.goto(17,50)
+jack.pendown()
+jack.fill(True)
+jack.circle(4)
+jack.fill(False)
+jack.penup()
+jack.goto(-17,50)
+jack.pendown()
+jack.fill(True)
+jack.circle(4)
+jack.fill(False)
+#Jaw
+ell.pensize(2)
+ell.goto(38,45)
+ell.pendown()
+ell.goto(38,10)
+ell.goto(20,-5)
+ell.penup()
+ell.goto(-37,45)
+ell.pendown()
+ell.goto(-38,10)
+ell.goto(-20,-5)
+ell.penup()
+ell.goto(-7,-10)
+ell.pendown()
+ell.goto(-5,-13)
+ell.goto(5,-13)
+ell.goto(7,-10)
+ell.penup()
+#Sideburns
+jack.penup()
+jack.goto(37,45)
+jack.pendown()
+jack.fill(True)
+jack.goto(41,45)
+jack.goto(41,60)
+jack.goto(37,60)
+jack.goto(37,45)
+jack.fill(False)
+jack.penup()
+jack.goto(-37,45)
+jack.pendown()
+jack.fill(True)
+jack.goto(-41,45)
+jack.goto(-41,60)
+jack.goto(-37,60)
+jack.goto(-37,45)
+jack.fill(False)
+jack.penup()
+#Ears
+ell.goto(42,45)
+ell.pendown()
+ell.fill(True)
+ell.goto(42,60)
+ell.goto(43,62)
+ell.goto(45,63)
+ell.goto(47,62)
+ell.goto(48,60)
+ell.goto(48,45)
+ell.goto(47,43)
+ell.goto(45,42)
+ell.goto(43,43)
+ell.goto(42,45)
+ell.fill(False)
+ell.penup()
+ell.goto(-42,45)
+ell.pendown()
+ell.fill(True)
+ell.goto(-42,60)
+ell.goto(-43,62)
+ell.goto(-45,63)
+ell.goto(-47,62)
+ell.goto(-48,60)
+ell.goto(-48,45)
+ell.goto(-47,43)
+ell.goto(-45,42)
+ell.goto(-43,43)
+ell.goto(-42,45)
+ell.fill(False)
+ell.penup()
+#Hat
+jack.speed(0)
+jack.goto(37,60)
+jack.fill(True)
+jack.goto(80,60)
+jack.goto(80,65)
+jack.goto(37,65)
+jack.goto(32,95)
+jack.goto(-32,95)
+jack.goto(-37,65)
+jack.goto(-80,65)
+jack.goto(-80,60)
+jack.goto(37,60)
+jack.fill(False)
+jack.penup()
+jack.goto (0,-150)
+jack.penup()
+```
+
```

Comment:
Phew!  I do like it when students include code in a reflection, but you might consider pulling out just the pieces that are most relevant.  These long code blocks can be difficult to read.